static/js/util/currency.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import cc from 'currency-codes';
+import R from 'ramda';
+
+const codeToOption = code => (
+  { value: code, label: cc.code(code).currency }
+);
+
+const labelSort = R.sortBy(R.compose(R.toLower, R.prop('label')));
+
+const excludeUnusedCodes = R.reject(R.flip(R.contains)(['USS', 'USN']));
```

Comment:
oh yeah, they are alternative codes for `USD`. `USN` is united states dollar next-day funds, and I think `USS` is some variant of that.